modules/renderer.js

```diff
@@ -262,16 +271,16 @@ class Heebo extends Font {
  * Returns a string of the current time string.
  */
 function curTimeString() {
```

Comment:
lots of lets needed

Reply:
(FYI: var generally yields better performance than let in a lot of cases... not that it matters in this application.)